x/bep3/types/genesis.go

```diff
@@ -29,16 +21,16 @@ func NewGenesisState(params Params, swaps AtomicSwaps, supplies AssetSupplies, p
 func DefaultGenesisState() GenesisState {
 	return NewGenesisState(
 		DefaultParams(),
-		AtomicSwaps{},
-		AssetSupplies{},
+		[]AtomicSwap{},
+		[]AssetSupply{},
 		DefaultPreviousBlockTime,
 	)
 }
 
 // Equal checks whether two GenesisState structs are equivalent.
 func (gs GenesisState) Equal(gs2 GenesisState) bool {
```

Comment:
i believe we can just remove the `Equal` func since they are not used anywhere. also allows us to not used amino here